circle.yml

```diff
@@ -1,12 +1,34 @@
+general:
+  artifacts:
+  - ~/drafter/emcc/jsdrafter.js
+  - ~/drafter/emcc/jsdrafter.js.mem
```

Comment:
No there is no place to download them, but can be easily done and I suppose that any `drafter` release will have a `drafter.js` company.